### PR TITLE
test for bond with sub vlans

### PR DESCRIPTION
**Pull request by SchSeba**
This PR add a test and also an example document for creating a bond interfaces with sub vlan interfaces.

for example you can create a bond0 interface and then add one sub vlan interface bond0.102 that will tag packets on vlan 102 via that bond.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/nmstate/kubernetes-nmstate/288)

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by phoracek** [comment]
ci add to whitelist

File: docs/user-guide-policy-configure-linux-bond-with-vlans.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+with slaves `eth1` and `eth2` and one vlan interface with `bond0` as parent interface
+
+## Requirements
+
+Before we start, please make sure that you have your Kubernetes/OpenShift
+cluster ready. In order to do that, you can follow the guides of deployment on
+[local cluster](deployment-local-cluster.md) or your
+[arbitrary cluster](deployment-arbitrary-cluster.md).
+
+## Configure bond
```
Comment (phoracek) [inline docs/user-guide-policy-configure-linux-bond-with-vlans.md]: bond and vlan

File: docs/user-guide-policy-configure-linux-bond-with-vlans.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+All you have to do in order to create the bond and the sub vlan interface on all nodes across cluster is
+to apply the following policy:
+
+```yaml
+cat <<EOF | ./kubevirtci/cluster-up/kubectl.sh create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond0-eth1-eth2-policy
```
Comment (phoracek) [inline docs/user-guide-policy-configure-linux-bond-with-vlans.md]: mention vlan in the name

File: docs/user-guide.md [hunk 3]
```diff
@@ -11,3 +11,4 @@ the tutorials.
 - [Create a Linux bridge and connect it to a node interface](user-guide-policy-configure-linux-bridge.md)
 - [Create an Open vSwitch bridge and connect it to a node interface](user-guide-policy-configure-ovs-bridge.md)
 - [Configure a Linux bonding interface](user-guide-policy-configure-linux-bond.md)
+- [Configure a Linux bonding interface with sub vlan interfaces](user-guide-policy-configure-linux-bond-with-vlans.md)
```
Comment (phoracek) [inline docs/user-guide.md]: What does "sub" mean? Can we make it with "VLAN interface"

**Comment by SchSeba** [comment]
@phoracek thanks for the review I done with the comments can you please take another look?

**Pull request merged**